components/mosart/src/wrm/WRM_type_mod.F90

```diff
@@ -78,6 +79,7 @@ MODULE WRM_type_mod
      real(r8), pointer :: MaxStorTarget(:)  ! (nd)
 
      integer , pointer :: YEAR(:)           ! (nd) year dam was constructed and operationnal
+     integer , pointer :: active(:)         ! (nd) whether dam is not functional (before construction) or during filling (<10 yrs after built and <80% of capacity) or fully functional (10 yrs after built or >80% of capacity) 
```

Comment:
I would have guessed `active` will be either `0` or `1`, but it could have the following values:
- `0`: Not functional
- `1`: Filling
- `2`: Fully functional

How about renaming `active` to indicate stages (e.g. `activity_stage`)?

Reply:
Thanks for the suggestion. Sure I can do it.